hspec.cabal

```diff
@@ -32,7 +32,7 @@ description:      Hspec is a testing framework for Haskell. It is inspired by
                   The Hspec Manual is at <http://hspec.github.io/>.
 
 extra-source-files:
-    changelog
+    CHANGES.markdown
```

Comment:
`hspec.cabal` is generated from `package.yaml` via [sol/hpack](https://github.com/sol/hpack). So this change should go into `package.yaml`, after that you can re-generated `hspec.cabal` by simply running:


```
$ hpack
```